tools/lint-js.js

```diff
@@ -16,12 +16,11 @@ const glob = require('./eslint/node_modules/glob');
 const cwd = process.cwd();
 const cliOptions = {
   rulePaths: rulesDirs,
-  extensions: extensions,
+  extensions: extensions
 };
 
 // Check if we should fix errors that are fixable
-if (process.argv.indexOf('-F') !== -1)
-  cliOptions.fix = true;
+if (process.argv.indexOf('-F') !== -1) cliOptions.fix = true;
```

Comment:
-1 I think these kinds of changes make things less readable.

Reply:
I apologize I think prettier made these changes automatically.

Reply:
Ah ok. There are other instances of this in this file that should also be reverted.